modules/core/shared/src/main/scala/lucuma/core/model/SiteCoordinatesLimits.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.all.*
+import lucuma.core.math.Coordinates
+import lucuma.core.math.Declination
+import lucuma.core.math.RightAscension
+
+case class SiteCoordinatesLimits(
+  raStart:  RightAscension,
+  raEnd:    RightAscension,
+  decStart: Declination,
+  decEnd:   Declination
+) derives Eq {
+  def inRaLimits(ra: RightAscension): Boolean =
+    if (raStart < raEnd)
+      raStart <= ra && ra <= raEnd
+    else
+      raStart <= ra || (ra.toHourAngle.toDoubleDegrees >= 0 && ra <= raEnd)
```

Comment:
Is the `0` check necessary?  Seems like an RA should always be `[0, 24)`.

Reply:
I took this from the phase1 model code but I was wondering the same as I typed.
I can take it away

Reply:
It seems like we should include an assertion in the `HourAngle` "constructor", but all ways to make an `HourAngle` appear to produce a result that is in range.